script/licenses.py

```diff
@@ -189,61 +205,250 @@ def from_dict(cls, data: dict[str, str]) -> PackageDefinition:
     "uvcclient": AwesomeVersion(
         "0.11.0"
     ),  # No License https://github.com/kk7ds/uvcclient/issues/7
+    # -- Full license text in metadata
```

Comment:
Should these be in TODO or in exceptions?

Reply:
All of them are OSI licenses, so I wouldn't consider them to be "exceptions" per se. More things we should take care of which would be "todo".

I can change it though if you think it makes more sense there.